public/assets/homepage.js

```diff
@@ -192,4 +197,71 @@ $( document ).ready(function() {
     $(".bodyStartBtn").on("click", function(){
         logWebpageActivity("Click_module=StartExploring_location=Index");
     });
+
+    // Setup video lazyPlay
+    $(window).on("scroll", onScroll);
+
+    vidBanner = $('#vidbanner')[0];
+    bannerVid = $('#bgvid')[0];
+
+    instructVideoContainer = $('#instructionvideo')[0];
+    instructVideos = [$('#vid1')[0],
+        $('#vid2')[0],
+        $('#vid3')[0]];
+
+    // Auto advance instruction videos
+    switchToVideo(DEFAULT_VIDEO);
+    setInterval(autoAdvanceLaptopVideos, 500);
 });
+
+var pausedVideos = {};
+
+// Wrappers around lazyPlayVideos()
+var lazyPlayVideosThrottled = _.throttle(lazyPlayVideos, 300);
+var lazyPlayVideosDebounced = _.debounce(lazyPlayVideos, 600);
+
+// Triggered when the user scrolls
+function onScroll() {
+    lazyPlayVideosThrottled(); // While scrolling, run the check every 300ms
+    lazyPlayVideosDebounced(); // After scrolling, make sure we run the check
+}
+
+// lazyPlays our main videos
+function lazyPlayVideos() {
+    lazyPlay(vidBanner, bannerVid);
+    return;
+    for(var i = 0; i < instructVideos.length; i++) {
```

Comment:
I would keep a space between the `for` and the `(`. In general, I would suggest having a space before an open paren _unless_ it is a function call. So `for (...)` and `if (...)`, but `lazyPlay(...)` and `myFunc(...)`.

Reply:
I also forgot to remove the "return;" there.